db/hints/internal/hint_endpoint_manager.cc

```diff
@@ -196,6 +196,7 @@ future<db::commitlog> hint_endpoint_manager::add_store() noexcept {
         return io_check([name = _hints_dir.c_str()] { return recursive_touch_directory(name); }).then([this] () {
             commitlog::config cfg;
 
+            cfg.sched_group = _shard_manager.local_db().commitlog()->active_config().sched_group;
```

Comment:
Am I right that before this patch hints commitlog was using default sched group?

Reply:
Yes